src/views/pages/network.html

```diff
@@ -21,22 +21,6 @@
 		{{/each}}
 	</style>
 
-	{{#if !@shared.isServer && !@shared.googleMapsLoaded}}
```

Comment:
There's no difference here, the scripts are already dynamically inserted and therefore async. Please revert this part.

Reply:
Done! Out of curiosity where is the dynamic insertion happening from?

Reply:
The whole file is a Ractive component so the rendering is handled by Ractive and this section is specifically restricted from server rendering (`!@shared.isServer`) so it happens on the client.

Reply:
Thank you! I'll look into how Mustaches is used, I haven't come across it before